src/test/java/com/google/cloud/tools/maven/RunAsyncMojoTest.java

```diff
@@ -40,13 +46,22 @@
   @InjectMocks
   private RunAsyncMojo runAsyncMojo;
 
+  @Before
+  public void setUp(){
+    MockitoAnnotations.initMocks(this);
+  }
+
   @Test
-  public void testRunAsync() throws MojoFailureException, MojoExecutionException, IOException {
+  @Parameters({"1,V1", "2-alpha,V2ALPHA"})
+  public void testRunAsync(String version, SupportedVersion mockVersion)
+      throws MojoFailureException, MojoExecutionException, IOException {
     final int START_SUCCESS_TIMEOUT = 25;
 
     // wire up
+    runAsyncMojo.devserverVersion = version;
```

Comment:
But the two values are the same, no? Shouldn't we be able to derive one from the other? Or is that bad for this test?